parselxx.py

```diff
@@ -45,15 +68,19 @@ def parse_file(filename):
 			word_nr = word_nr + 1
 			count = count + 1
 			a = re.split("\s+", line)
-			word = a[0] # word is first in array
-			encoded_word = str.encode(betacode.transliterate(word))
-			unicode_word = encoded_word.decode('utf-8')
-			sqlcode = "INSERT INTO content VALUES (\'" + str(count) + "\', \'" + book_name + "\', \'" + str(chapter_nr) + "\', \'" + str(verse_nr) + "\', \'" +  str(word_nr) + "\', \'" + unicode_word + "\')"
-			#outfile.write(unicode_word.encode('utf-8'))
-			dbconn.execute(sqlcode)
-			dbconn.commit()
+			word = get_unicode_word(a[0])
+			root_word = get_unicode_word(a[-1])
+			valuesToInsert.append("(\'" + str(count) + "\', \'" + book_name + "\', \'" + str(chapter_nr) + "\', \'" + str(verse_nr) + "\', \'" +  str(word_nr) + "\', \'" + word + "\', \'" + root_word + "\')")
+			chunkCounter += 1
+
+		if (chunkCounter == 499):
```

Comment:
Just curious, how did you choose the number 499? It would be clearer if it was set to a const variable.